<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
      <item>
       <widget class="QImageWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="curentlblLabel">
           <property name="palette">
            <palette>
             <active>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>238</red>
                 <green>238</green>
                 <blue>236</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>238</red>
                 <green>238</green>
                 <blue>236</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>120</red>
                 <green>120</green>
                 <blue>120</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>548</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="statusLabel">
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,10,1,0,10,1,1">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="radius_label">
               <property name="text">
                <string>Brush radius:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="radius_value_label">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>1</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QSlider" name="radius_slider">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>20</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
             <property name="tickInterval">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>518</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Labels intensity:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="intensitySlider">
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Keyboard:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>    →  - next file</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>    ←  - previous file</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Mouse:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>    right     - draw</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>    middle - translation</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>    left        - context menu</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>    wheel   - zoom</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1141</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionactionopenDirectory"/>
    <addaction name="actionEqualizeImage"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionShowAbout"/>
   </widget>
   <addaction name="menuActions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCopyPointsToClipboard">
   <property name="text">
    <string>Copy points to clipboard</string>
   </property>
   <property name="toolTip">
    <string>Copy points to clipboard</string>
   </property>
  </action>
  <action name="actionclearPoints">
   <property name="text">
    <string>Очистить всё</string>
   </property>
   <property name="toolTip">
    <string>Clear points</string>
   </property>
  </action>
  <action name="actionShowAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>Show about</string>
   </property>
  </action>
  <action name="actionLoadPoints">
   <property name="text">
    <string>Load points</string>
   </property>
   <property name="toolTip">
    <string>Load points from user input</string>
   </property>
  </action>
  <action name="actionactionopenDirectory">
   <property name="text">
    <string>Open directory</string>
   </property>
   <property name="toolTip">
    <string>Open directory for markup</string>
   </property>
  </action>
  <action name="actionEqualizeImage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Histogram equalization</string>
   </property>
   <property name="toolTip">
    <string>Apply histogram equalization before show</string>
   </property>
  </action>
  <action name="actionFlipImageVertically">
   <property name="text">
    <string>Flip vertically</string>
   </property>
   <property name="toolTip">
    <string>Flip image vertically</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionFlipImageHorizontally">
   <property name="text">
    <string>Flip horizontally</string>
   </property>
   <property name="toolTip">
    <string>Flip image horizontally</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionDeleteImage">
   <property name="text">
    <string>Delete image</string>
   </property>
   <property name="toolTip">
    <string>Delete image</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionRotateImage180">
   <property name="text">
    <string>Rotate 180</string>
   </property>
   <property name="toolTip">
    <string>Rotate image 180 degrees</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionRotateImage90">
   <property name="text">
    <string>Rotate 90</string>
   </property>
   <property name="toolTip">
    <string>Rotate image 90 degrees</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionRotateImageMinus90">
   <property name="text">
    <string>Rotate -90</string>
   </property>
   <property name="toolTip">
    <string>Rotate image -90 degrees</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actioneraser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/eraser.svg</normaloff>:/Resources/eraser.svg</iconset>
   </property>
   <property name="text">
    <string>Стереть</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionlabel_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1. Плечеголовной ствол</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionlabel_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2. Левая общая сонная артерия</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionlabel_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3. Левая подключичная артерия</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionlabel_4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1.1. Нижняя диафрагмальная артерия левая</string>
   </property>
   <property name="toolTip">
    <string>1. Нижняя диафрагмальная артерия левая</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionlabel_5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2. Чревный ствол</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionlabel_6">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3. Левая желудочковая артерия</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionlabel_7">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>4. Общая печеночная артерия</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionlabel_8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5. Селезеночная артерия</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionlabel_9">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>6. Верхняя брыжеечная артерия</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionlabel_10">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>7. Нижняя брыжеечная артерия</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionlabel_11">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>8.1. Почечная артерия левая</string>
   </property>
   <property name="toolTip">
    <string>8. Почечная артерия левая</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionlabel_12">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>9. Нижняя брыжеечная артерия</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionlabel_13">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>10.1. Общая подвздошная артерия левая</string>
   </property>
   <property name="toolTip">
    <string>10.1. Общая подвздошная артерия левая</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionlabel_14">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1. Дуга аорты</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionlabel_15">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2. Нисходящий отдел аорты</string>
   </property>
   <property name="toolTip">
    <string>2. Нисходящий отдел аорты</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionlabel_16">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3. Брюшной отдел аорты</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionaneurysm">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/aneurysm.svg</normaloff>:/Resources/aneurysm.svg</iconset>
   </property>
   <property name="text">
    <string>Аневризма</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionlabel_17">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1.2. Нижняя диафрагмальная артерия правая</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionlabel_18">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>10.2. Общая подвздошная артерия правая</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionlabel_19">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>8.2 Почечная артерия правая</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QImageWidget</class>
   <extends>QWidget</extends>
   <header>qimagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>radius_slider</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
